### ISSUE TITLE
Extra Utilities 2 - 1.12 - 1.5.1 - Mining Dimension Entities Spam

### ISSUE THREAD

**Issue by xentripetal** [rwtema/extrautilities_old_issues#1636]
[RAW]                                        |       E |     TE      |    C | 
 ExtraUtils2_Quarry_Dim/-9999 | 1300 | 128521 | 0    |

Quantum quarries are generating hundreds of thousands of tile entities that are causing lag. I am forced to put a tick time limiter on the dimension just to keep the server running.

**Comment by OldDogy** [comment]
The Quarry is bugged, and all current issues that the people are reporting are actually one same issue.
I went to the quarry world to watch what happens, and made this small documentation of pictures.

http://imgur.com/a/NjFtn

When you start it up everything is fine, it claims a 3x3chunk area and starts mining the middle chunk. when that is done, it will eat ur filter and everything in it, and start regenerating all 8 chunks around it but not the middle one that it's actually mining.
This leads to the quarry creating hundreds or thousands of items on the floor saplings , and seeds of all kinds. because it's constantly regenerating the 8 chunks spawning in new trees and ores and grass in it.
everything that you start getting from the quarry after the first chunk is done is flowers and saplings.. 

**I'm not sure If RWTema is even active here anymore ? Issues are just stacking in here and no one seems to be looking at it. Not to mention that this is actually extrautilities 1 issue tracker, and hasn't been looked at since 2014-15**

**Comment by Thk54** [comment]
here were my findings about the Quarry https://github.com/rwtema/extrautilities/issues/1601 its doing the same thing to all 9 chunks, its just that it has no blocks to gen things into on the middle chunk

**Comment by morerokk** [comment]
Yeah, I'm noticing a lot of issues with the Quantum Quarry too. It just seems to cause errors every now and then, which eats the filters inside and spams the server console.
